### PR TITLE
Use system ECDsa

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: src/neo/Cryptography/Crypto.cs [hunk 1]
```diff
@@ -35,74 +34,26 @@ public static byte[] Sign(byte[] message, byte[] prikey, byte[] pubkey)
 
         public static bool VerifySignature(ReadOnlySpan<byte> message, ReadOnlySpan<byte> signature, ECC.ECPoint pubkey)
         {
-            if (pubkey.Curve == ECC.ECCurve.Secp256r1)
+            ECCurve curve =
+                pubkey.Curve == ECC.ECCurve.Secp256r1 ? ECCurve.NamedCurves.nistP256 :
+                pubkey.Curve == ECC.ECCurve.Secp256k1 ? ECCurve.CreateFromOid(new Oid("1.3.132.0.10")) :
```
Comment (shargon) [inline src/neo/Cryptography/Crypto.cs:39]: Is this curve enabled for all operative systems?

Comment (erikzhang) [reply src/neo/Cryptography/Crypto.cs:39]: I'm not sure. Can you test it on linux?

Comment (shargon) [reply src/neo/Cryptography/Crypto.cs:39]: Yes, It works on github actions, and it was in linux

```suggestion
                pubkey.Curve == ECC.ECCurve.Secp256k1 ? ECCurve.CreateFromFriendlyName("secP256k1") :
```

Comment (erikzhang) [reply src/neo/Cryptography/Crypto.cs:39]: Did you test this? I think there is no difference.

Comment (shargon) [reply src/neo/Cryptography/Crypto.cs:39]: Yes, I tested with `secP256k1`, It's the same but more readable

**Review by shargon — APPROVED** [review]

**Comment by erikzhang** [comment]
@neo-project/ngd-shanghai Can you test more os to see whether it works?

**Review by vncoelho — APPROVED** [review]
Worked here as well, however, still Linux based versions.

**Comment by ProDog** [comment]
> @neo-project/ngd-shanghai Can you test more os to see whether it works?

OK. I'll test.

**Comment by ProDog** [comment]
I have tested with multiple nodes on win10 and linux, it's working well.

**Comment by erikzhang** [comment]
Did you test Ubuntu and CentOS?

**Comment by ProDog** [comment]
> Did you test Ubuntu and CentOS?

Yes, the linux nodes includes Ubuntu and CentOS.

**Pull request merged**